src/cast_core/src/actors/RunCommand.cpp

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <cast_core/actors/RunCommand.hpp>
```

Comment:
So usually you include the declaration header first before system headers because it makes the include order more predictable. (This is suggested in the google style guide for instance.)